tests/inventory.yaml.j2

```diff
@@ -1 +1,13 @@
-{{ { 'all': { 'hosts': hostvars } } | to_yaml }}
+{% set role_vars = vars.keys() | intersect((
```

Comment:
should be named `inventory.yml.j2` - the system roles best practices is to use `.yml` instead of `.yaml`